src/components/expanded-state/swap-settings/SwapSettingsState.js

```diff
@@ -96,6 +98,12 @@ export default function SwapSettingsState({ asset }) {
     updateSource(Source.AggregatorRainbow);
   }, [settingsChangeFlashbotsEnabled, updateSource]);
 
+  const openExplainer = () => {
```

Comment:
this comment is for line 84 

![image](https://user-images.githubusercontent.com/12115171/175655294-45eed192-5fa6-4ed0-a744-ee267c2cf958.png)

since we have a title now, can we use 

```
(android ? 275 : 245) + (swapSupportsFlashbots ? 55 : 0);
```
with this change it will look like
![image](https://user-images.githubusercontent.com/12115171/175655459-506252e7-cb0f-4912-a3c1-45f8e2b9d636.png)

![image](https://user-images.githubusercontent.com/12115171/175655491-fb9cae12-7473-470a-b6a7-bcac47c769f8.png)